tests/conftest.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import httpx
+from unittest import mock
+
+# from contextlib import asynccontextmanager
+import pytest_asyncio
+from fakeredis import FakeStrictRedis
+from fastapi.testclient import TestClient
+from server.app import _get_redis_conn
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--use-openai",
+        action="store_true",
+        default=False,
+        help="run tests that require OpenAI key",
+    )
+    parser.addoption(
+        "--use-server",
+        action="store_true",
+        default=False,
+        help="run tests that require running adala server",
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "use_openai: mark test as requiring OpenAI key")
```

Comment:
why not just add these lines to pyproject.toml?

Reply:
either way would work, I just thought it was more discoverable to keep all the pytest stuff together in one file